File: listas-logpp/lista 2/lista 2 sala/questao 5.py
A = float(input("Digite o primeiro valor:"))
B = float(input("Digite o segundo valor:"))
C = float(input("Digite o terceiro valor:"))
if A>B>C:
    print(f"    {A}     {B}      {C}")
elif A>C>B:
    print(f"    {A}     {C}      {B}")
elif B>A>C:
    print(f"    {B}     {A}      {C}")
elif B>C>A:
    print(f"    {B}     {C}      {A}")
elif C>A>B:
    print(f"    {C}     {A}      {B}")
elif C>B>A:
    print(f"    {C}     {B}      {A}")

File: listas de exercicio/lista 2/lista 2 sala C/questao 5.c
#include <stdio.h>

int main(){
    int A, B, C;
    printf("Digite o primeiro valor:");
    scanf("%d", &A);
    printf("Digite o segundo valor:");
    scanf("%d", &B);
    printf("Digite o terceiro valor:");
    scanf("%d", &C);

    if (A>B && B>C){
        printf("A ordem crescente é:%d, %d, %d", C, B, A);
    } else if (B>A && A>C){
        printf("A ordem crescente é:%d, %d, %d", C, A, B);
    } else if (C>B && B>A){
        printf("A ordem crescente é:%d, %d, %d", A, B, C);
    } else if (A>C && C>B){
        printf("A ordem crescente é:%d, %d, %d", B, C, A);
    } else if (B>C && C>A){
        printf("A ordem crescente é:%d, %d, %d", A, C, B);
    } else if (C>A && A>B){
        printf("A ordem crescente é:%d, %d, %d", B, A, C);
    }
}